grafana-cloud-metrics.yaml

```diff
@@ -118,14 +118,14 @@ outputs:
 
 resources:
   wait_condition:
-    type: "OS::Heat::SwiftSignal"
+    type: "AWS::CloudFormation::WaitCondition"
```

Comment:
@rs-randallburt I did not know this. We had some issues with signals that didn't fire when using `SwiftSignal`, the AWS wait condition appeared to be more stable.